src/Rings/mpoly-affine-algebras.jl

```diff
@@ -10,27 +26,88 @@ export isreduced
 @doc Markdown.doc"""
     isreduced(A::MPolyQuo)
 
-Return `true` if `A` is reduced, `false` otherwise.
+Returns `true` if `A` is reduced, `false` otherwise.
 """
-function isreduced(A::MPolyQuo{T}) where T
+function isreduced(A::MPolyQuo) 
   I = A.I
   return I == radical(I)
 end
 
+@doc Markdown.doc"""
+    isnormal(A::MPolyQuo)
+"""
+function isnormal(A::MPolyQuo) 
+  aaa
```

Comment:
Do you plan to implement these before this PR is merged? Otherwise, you can just do something like this as a "safe" placeholder...
```suggestion
  error("TODO: implement this")
```